app/kuma-dp/pkg/dataplane/metrics/testdata/otel/counter.golden.json

```diff
@@ -86,7 +86,7 @@
             }
           ],
           "StartTime": "0001-01-01T00:00:00Z",
-          "Time": "0001-01-01T00:00:00Z",
+          "Time": "2024-01-01T01:01:01.000000001Z",
```

Comment:
this is fallback / request time from https://github.com/kumahq/kuma/pull/10966/files#r1689914172